packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -162,27 +165,27 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   int get hashCode => dartDiagnosticRef.hashCode;
 
   /// Separator text to show between property names and values.
-  String get separator => showSeparator ? ':' : '';
+  String get separator => showSeparator! ? ':' : '';
 
   /// Label describing the [RemoteDiagnosticsNode], typically shown before a separator
   /// (see [showSeparator]).
   ///
   /// The name should be omitted if the [showName] property is false.
-  String get name => getStringMember('name');
+  String? get name => getStringMember('name');
 
   /// Whether to show a separator between [name] and description.
   ///
   /// If false, name and description should be shown with no separation.
   /// `:` is typically used as a separator when displaying as text.
-  bool get showSeparator => getBooleanMember('showSeparator', true);
+  bool? get showSeparator => getBooleanMember('showSeparator', true);
```

Comment:
non nullable since a default is provided

Reply:
done